### PR TITLE
[Feature/dashboardComponent] Add dashboard component

### PR DESCRIPTION
**Pull request by AlanAnguiano**
## Description

Includes the dashboard component as well as a change style in the navbar while Navigate between options or views

## Evidence

image
image

## Link Issue (if applicable)
  -

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -8,11 +8,18 @@
...
 * {
   margin: 0;
   padding: 0;
-}
\ No newline at end of file
+}
+
+a {
+  text-decoration: none;
+  color: #000
```
Comment (Usiel-03A7) [inline src/index.css]: sort alphabetically.

**Review by AlejandroCuriel — DISMISSED** [review]
LGTM 🚀

**Review by Usiel-03A7 — COMMENTED** [review]
just leaving some suggestions.

**Review by Usiel-03A7 — APPROVED** [review]

**Review by AlejandroCuriel — APPROVED** [review]
LGTM 🚀👌

**Pull request merged**